docs/.editorconfig

```diff
@@ -0,0 +1,11 @@
+# editorconfig.org
```

Comment:
This file is not editor specific, but it is read by most recent editors, it defines the guideline when editing files.

For example, this file is telling the editor that he should use 4 spaces to indent, he should add a new line and should use LR instead of CRLF.

It is useful when one what a single style to be applied to all the files present in a folder.

More at: https://editorconfig.org/

Reply:
And even LEGACY ;-) editors can use it (https://marketplace.eclipse.org/content/editorconfig-eclipse)